python/py-avro/Portfile

```diff
@@ -18,17 +19,36 @@ long_description    ${description}
 
 homepage            https://avro.apache.org/
 
-checksums           rmd160  61c45d1cfa9914306401cb88d880ac63602d43cc \
-                    sha256  f456539183df0d6912daedfbf0f94d7cf69b4fba7200490e8a06d1e8f23af43d \
-                    size    1845163
+checksums           rmd160  e398cc448a227de8754eb60848eb4a47f0633265 \
+                    sha256  4e94b43ebdb1081fd842644bda2caaddc586c1b608efe9236d348d01bf0866a4 \
+                    size    1899682
 
 python.versions     37 38
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                        port:py${python.version}-setuptools
-
-    worksrcdir          ${distname}/lang/py3
+                    port:py${python.version}-setuptools
```

Comment:
the package doesn't build since you didn't include all dependencies - please start building things locally in trace-mode (using the `-v` flag) which will show you these things before submitting a PR

Reply:
Show where? I don't see how this shows missing dependencies.

Here's what I see:

> `sudo port -v build py37-avro`
```
--->  Computing dependencies for py37-avro.
--->  Fetching distfiles for py37-avro
Cloning into '/opt/local/var/macports/build/_opt_local_ports_python_py-avro/py37-avro/work/avro-1.10.0'...
remote: Enumerating objects: 82, done.        
remote: Counting objects: 100% (82/82), done.        
remote: Compressing objects: 100% (77/77), done.        
remote: Total 50076 (delta 35), reused 38 (delta 4), pack-reused 49994        
Receiving objects: 100% (50076/50076), 16.87 MiB | 19.56 MiB/s, done.
Resolving deltas: 100% (26070/26070), done.
--->  Verifying checksums for py37-avro
--->  Extracting py37-avro
--->  Configuring py37-avro
--->  Building py37-avro
Executing:  cd "/opt/local/var/macports/build/_opt_local_ports_python_py-avro/py37-avro/work/avro-1.10.0/lang/py3" && /opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 setup.py --no-user-cfg build -j12 
running build
running build_py
creating build
creating build/lib
creating build/lib/avro
copying avro/datafile.py -> build/lib/avro
copying avro/ipc.py -> build/lib/avro
copying avro/protocol.py -> build/lib/avro
copying avro/io.py -> build/lib/avro
copying avro/__init__.py -> build/lib/avro
copying avro/schemanormalization.py -> build/lib/avro
copying avro/tool.py -> build/lib/avro
copying avro/schema.py -> build/lib/avro
running egg_info
creating avro_python3.egg-info
writing avro_python3.egg-info/PKG-INFO
writing dependency_links to avro_python3.egg-info/dependency_links.txt
writing requirements to avro_python3.egg-info/requires.txt
writing top-level names to avro_python3.egg-info/top_level.txt
writing manifest file 'avro_python3.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'avro_python3.egg-info/SOURCES.txt'
copying avro/HandshakeRequest.avsc -> build/lib/avro
copying avro/HandshakeResponse.avsc -> build/lib/avro
copying avro/LICENSE -> build/lib/avro
copying avro/NOTICE -> build/lib/avro
copying avro/VERSION.txt -> build/lib/avro
creating build/lib/avro/tests
copying avro/tests/av_bench.py -> build/lib/avro/tests
copying avro/tests/gen_interop_data.py -> build/lib/avro/tests
copying avro/tests/run_tests.py -> build/lib/avro/tests
copying avro/tests/sample_http_client.py -> build/lib/avro/tests
copying avro/tests/sample_http_server.py -> build/lib/avro/tests
copying avro/tests/test_datafile.py -> build/lib/avro/tests
copying avro/tests/test_datafile_interop.py -> build/lib/avro/tests
copying avro/tests/test_enum.py -> build/lib/avro/tests
copying avro/tests/test_init.py -> build/lib/avro/tests
copying avro/tests/test_io.py -> build/lib/avro/tests
copying avro/tests/test_ipc.py -> build/lib/avro/tests
copying avro/tests/test_normalization.py -> build/lib/avro/tests
copying avro/tests/test_protocol.py -> build/lib/avro/tests
copying avro/tests/test_schema.py -> build/lib/avro/tests
copying avro/tests/test_script.py -> build/lib/avro/tests
running build_scripts
creating build/scripts-3.7
copying and adjusting scripts/avro -> build/scripts-3.7
changing mode of build/scripts-3.7/avro from 644 to 755
```

Reply:
sorry, my bad to use trace-mode you'll have of course to use the `-t` flag to make sure it will only use dependencies that you actually specified.

Reply:
Oh I see:
> `sudo port -vst build py37-avro`

Reply:
yes, as the PR checklist tells you to do...